ckanext/restricted/controller.py

```diff
@@ -85,6 +85,35 @@ def _send_request_mail(self, data):
                 data.get('maintainer_email'): extra_vars.get('maintainer_name'),
                 extra_vars.get('admin_email_to'): '{} Admin'.format(extra_vars.get('site_title'))}
 
+            context = {
+                'model': model,
+                'session': model.Session,
+                'ignore_auth': True
+            }
+            dataset = toolkit.get_action('package_show')(
+                context, {'id': data.get('package_name')}
+            )
+            dataset_org = toolkit.get_action('organization_show')(
+                context, {'id': dataset['owner_org']}
+            )
+            dataset_org_admins = [
```

Comment:
please change the var name to `dataset_org_admin_ids`.
It's generally a bad practice to mutate variables if you don't have to (this var is reassigned a new value in :105)